pagefaultgames/pokerogue
# [HotFix][Sprite]  867 Runerigus cropped fix

Runerigus: 
- cropped static frames, fixed cropped sprite
- set runerigus exp to use the shiny exp's animation 
- verified all hex colors are unchanged

- fixed ultra necrozma exp front variant (swapped arrays.)
- xatu female eye color fix

## What are the changes the user will see?
- runerigus sprite no longer cropped or offset
- ultra necrozma's variant colors are no longer mismatched between front and back

## Why am I making these changes?
These are small fixes to the variant update.


### Screenshots/Videos


static:
default:
![e21134b5-8a01-413d-acfc-5ad717e3535f](https://github.com/user-attachments/assets/13ff24ba-5899-429d-934e-23b2c97eda6f)
shiny:
![0a369cfc-dcc5-452c-a34a-5a05634161b2](https://github.com/user-attachments/assets/b2394d3d-334b-4872-9cfd-81c630860f4b)
variant from shader:
![7bd1ed63-8459-4d07-b816-b86d876d745e](https://github.com/user-attachments/assets/6e53bb1b-f184-4494-95c1-dbcdd836f1e9)


animated (exp):
default:
![a783c721-eb4a-4f62-aeb5-14ff5174f372](https://github.com/user-attachments/assets/9079b5e1-a2de-46a4-896e-75e7dcf7961a)
shiny:
![f60f218d-6723-4557-8f1c-1173085bd59a](https://github.com/user-attachments/assets/56861508-4ed7-42bb-9cfd-629d5a4227f2)
variant from shader:
![d06fcf14-7c6b-4796-b9d8-5ace885d75d6](https://github.com/user-attachments/assets/3c533853-94c8-433d-a906-4dcc536b4667)


ultra necrozma T2 and T3 front exp fixed
![e81bb6c2-7693-4aef-9878-00c36cf673af](https://github.com/user-attachments/assets/896dcac7-e54c-4cdd-9772-a626759c09b3)



## How to test the changes?
runerigus : 867
to verify all, turn mixed animated on, and off

xatu: 178 female, 
set species to xatu, gender female, shiny true and  variant 1 or 2
eye should be same color as default xatu

ultra necrozma: 800, 
set animation to mixed animated
set user species to necrozma and form 3
start game, go to pokemon summary to see front sprite
variant should be the correct color

## Checklist
